spec/spec_helper.rb

```diff
@@ -43,7 +43,20 @@ def instantiate_fixture(druid, klass = ActiveFedora::Base)
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
   # Run each example in an ActiveRecord transaction
-  config.use_transactional_fixtures = true
+  config.use_transactional_fixtures = false
+
+  config.before :each do
+    if Capybara.current_driver == :rack_test
+      DatabaseCleaner.strategy = :transaction
+    else
+      DatabaseCleaner.strategy = :truncation
```

Comment:
Anything outside of the rack context will fail (I believe). So also poltergeist javascript tests will most likely also fail.